### PR TITLE
Fix font color for creature name

### LINKED ISSUE #152: Font color for creature name is incorrect
In original version of the game all creatures have gold color font while in current game it's standard white color:
image

Original game:
image

### PR DESCRIPTION
**Pull request by andrey-starodubtsev**
In the original game font color is yellow, not white.

This fixes #152.

### CODE REVIEW

File: src/fheroes2/dialog/dialog_armyinfo.cpp [hunk 1]
```diff
@@ -74,13 +74,13 @@ int Dialog::ArmyInfo(const Troop & troop, int flags)
...
     // name
-    text.Set(troop.GetName(), Font::BIG);
+    text.Set( troop.GetName(), Font::YELLOW_BIG );
     dst_pt.x = pos_rt.x + 140 - text.w() / 2;
     dst_pt.y = pos_rt.y + 40;
     text.Blit(dst_pt);
 
     // count
-    text.Set(GetString(troop.GetCount()));
+    text.Set( GetString(troop.GetCount()), Font::BIG );
```
Comment (andrey-starodubtsev) [inline src/fheroes2/dialog/dialog_armyinfo.cpp]: Font::BIG is used by default, by the way:

```
    Text(const std::string &, int ft = Font::BIG);
```

Comment (ihhub) [reply src/fheroes2/dialog/dialog_armyinfo.cpp]: Strange. At my Windows machine it was showing as yellow color with onpy your change. I'll check again at home.

Comment (ihhub) [reply src/fheroes2/dialog/dialog_armyinfo.cpp]: No, you're pointing to constructor, not to `Set` function.

**Comment by ihhub** [comment]
@andrey-starodubtsev thank you very much for your contribution to the project!

File: src/fheroes2/dialog/dialog_armyinfo.cpp (hunk 1 above)
Comment (andrey-starodubtsev) [reply src/fheroes2/dialog/dialog_armyinfo.cpp]: Yeah, missed the fact that the same `text` object is used for blitting both name and count and font must be restored; `Set` is overloaded, but it doesn't set font if it's omitted.

**Pull request merged**